src/query/interpret/awesome_memgraph_functions.cpp

```diff
@@ -1259,6 +1297,14 @@ std::function<TypedValue(const TypedValue *, int64_t, const FunctionContext &ctx
   if (function_name == "LOCALDATETIME") return LocalDateTime;
   if (function_name == "DURATION") return Duration;
 
+  const auto &maybe_found =
+      procedure::FindFunction(procedure::gModuleRegistry, function_name, utils::NewDeleteResource());
+
+  if (maybe_found) {
+    const auto *func = (*maybe_found).second;
+    return UserFunction(*func, function_name);
```

Comment:
Ohh, I forgot to comment on this one! Your forgot to save the ModulePtr. Without that it is not guaranteed that the module won't be unloaded. I think we have talked about this previously, but if you need help, let me know!